sw/host/opentitantool/opentitantool_derive/src/lib.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use proc_macro2::TokenStream;
+use quote::quote;
+use syn::{parse_macro_input, Data, DataEnum, DeriveInput, Fields, Ident, Variant};
```

Comment:
This is personal taste, but I like to write `syn::Data` and `syn::Fields`. *shruggle*

Reply:
I think if this proc-macro crate grows in complexity, it might be worthwhile to redo the names.  I added an item to the "Rust Reveiwer HowTo" doc to discuss this point.